js/initcontext.go

```diff
@@ -305,16 +305,44 @@ func (i *InitContext) Open(ctx context.Context, filename string, args ...string)
 	} else if isDir {
 		return nil, fmt.Errorf("open() can't be used with directories, path: %q", filename)
 	}
-	data, err := afero.ReadFile(fs, filename)
+	fileData, err := afero.ReadFile(fs, filename)
 	if err != nil {
 		return nil, err
 	}
 
-	if len(args) > 0 && args[0] == "b" {
-		ab := i.runtime.NewArrayBuffer(data)
+	if len(args) > 0 && contains(args[0], 'b') {
```

Comment:
I wonder if there is a particular reason why we changed string to rune? Or maybe we can stick with the string and use the `strings` package function?

Reply:
You're absolutely right: I had completely forgotten about the `strings.Contains*` methods. I'll use that instead indeed :)